app/konnect/configure/runtime-manager/runtime-groups/declarative-config.md

```diff
@@ -85,31 +109,27 @@ For this example, let's add a new service.
 the example with your generated UUID:
 
     ```yaml
-    _format_version: "0.1"
-    service_packages:
+    _format_version: "1.1"
+    _konnect:
+      runtime_group_name: default
+    services:
     - name: MyService
-      versions:
-      - implementation:
-          type: kong-gateway
-          kong:
-            service:
-              host: mockbin.org
-              port: 80
-              protocol: http
-              id: {YOUR_GENERATED_UUID}
-              routes:
-              - methods:
-                - GET
-                - POST
-                name: mockpath
-                paths:
-                - /mock
-                plugins:
-                - name: key-auth
-                  config:
-                    key_names:
-                    - apikey
-        version: "1"
+      host: mockbin.org
+      port: 80
+      protocol: http
+      id: {YOUR_GENERATED_UUID}
```

Comment:
This UUID is no longer necessary to set manually and is a leftover from ServiceHub config. Note to self to remove it.

Reply:
Also applies to step 1 of this section, which is for generating the UUID.